server/mailrepository/mailrepository-api/src/test/java/org/apache/james/mailrepository/api/MailRepositoryPropertiesStoreContract.java

```diff
@@ -28,12 +28,8 @@
 
 public interface MailRepositoryPropertiesStoreContract {
 
-    MailRepositoryProperties NON_BROWSABLE = MailRepositoryProperties.builder()
-        .browsable(false)
-        .build();
-    MailRepositoryProperties BROWSABLE = MailRepositoryProperties.builder()
-        .browsable(true)
-        .build();
+    MailRepositoryProperties NON_BROWSABLE = new MailRepositoryProperties(false);
+    MailRepositoryProperties BROWSABLE = new MailRepositoryProperties(true);
```

Comment:
I really prefer the builder based version, it makes more clear the meaning of true and false.